lisa/node.py

```diff
@@ -114,6 +116,23 @@ def support_sudo(self) -> bool:
 
         return self._support_sudo
 
+    def check_sudo_password_required(self) -> None:
```

Comment:
It needs to save the whole password prompt. It can be used to detect password needed next time, and used to remove it from stdout for consistent.

Reply:
Have added code to save the password prompt.